src/ui/SettingsButton.tsx

```diff
@@ -40,17 +40,21 @@ export default function SettingsCard(
     <>
       <Popover
         gutter={0}
+        closeOnBlur
         placement="bottom-start"
         isOpen={isOpen}
-        onOpen={open}
         onClose={close}
+        onOpen={open}
         autoFocus={true}
         preventOverflow
         strategy="fixed"
       >
         <PopoverTrigger>
           <Button
             onClick={open}
+            onMouseDown={(e: React.MouseEvent<HTMLButtonElement>) =>
```

Comment:
let's add a comment here describing why we've added this & link to the issues you found, so we don't forget why it's here

Reply:
Added!